### PR TITLE
Add @Child property for One-to-one relations (#74)

### PR DESCRIPTION
**Pull request by taikoo**
add Sources/FluentKit/Properties/Child.swift, edited from Sources/FluentKit/Properties/Children.swift:

see gist: https://gist.github.com/taikoo/5d92990f7baad57c9f2071aa908871df#file-child-patch

### CODE REVIEW

File: Sources/FluentKit/Properties/Child.swift [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+// MARK: Relation
+
+extension ChildProperty: Relation {
+    public var name: String {
+        "Child<\(From.self), \(To.self)>(for: \(self.parentKey))"
+    }
+
+    public func load(on database: Database) -> EventLoopFuture<Void> {
+        self.query(on: database).all().map {
```
Comment (0xTim) [inline Sources/FluentKit/Properties/Child.swift:135]: Why not just do `.first()` here instead?

**Review by 0xTim — CHANGES_REQUESTED** [review]
This will needs tests to test all the new functionality

File: Sources/FluentKit/Properties/Child.swift (hunk 1 above)
Comment (taikoo) [reply Sources/FluentKit/Properties/Child.swift:135]: OK, I'll fix this.

**Comment by kaishin** [comment]
Would be great to see this added. I'd have written tests but I am not familiar enough with fluent-kit to do so atm.

**Comment by gwynne** [comment]
This functionality was implemented in #419

**Pull request closed without merging**